### PR TITLE
chore(ntb): Merge ntb related changes on master

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

**Comment by pavlovicnemanja** [comment]
:+1:

File: app/scripts/translations.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
 angular.module('gettext').run(['gettextCatalog', function (gettextCatalog) {
 /* jshint -W100 */
 	gettextCatalog.setStrings('de', {"My Profile":"DE - My Profile"});
+    gettextCatalog.setStrings('en', {
```
Comment (petrjasek) [inline app/scripts/translations.js]: this part shouldn't go to master. we need to keep this project specific

File: scripts/superdesk-authoring/views/authoring-header.html [hunk 2]
```diff
@@ -138,10 +139,10 @@
             </div>
           </div>
         </div>
-
-        <div class="authoring-header__row" ng-if="metadata.subjectcodes && schema.subject">
+
+        <div class="authoring-header__row" ng-if="metadata.subjectcodes && schema.subject && !metadata.subjects">
```
Comment (petrjasek) [inline scripts/superdesk-authoring/views/authoring-header.html]: where/how is `metadata.subjects` set?

Comment (ioanpocol) [inline scripts/superdesk-authoring/views/authoring-header.html]: This is an alternative list of subjects code for ntb. If it is defined, the current one will not be showed.

File: app/scripts/translations.js [hunk 3]
```diff
@@ -1,5 +1,3 @@
 angular.module('gettext').run(['gettextCatalog', function (gettextCatalog) {
-/* jshint -W100 */
-	gettextCatalog.setStrings('de', {"My Profile":"DE - My Profile"});
```
Comment (petrjasek) [inline app/scripts/translations.js]: this should probably stay there otherwise it will break an e2e test

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**